### ISSUE TITLE
finetune影响loss收敛

### ISSUE THREAD

**Issue by tracycw** [zhunzhong07/person-re-ranking#11]
不finetune的基础上，loss一直不收敛，各种调参都没有效果
加上ResNet_50.caffemodel进行finetune，loss直线下降，这是啥情况

**Comment by zhunzhong07** [comment]
Without finetuning from Imagenet pretrained model, you should set a large initial learning rate, e.g. lr = 0.1
